python/lsst/obs/subaru/objectMasks.py

```diff
@@ -0,0 +1,132 @@
+import re
+import lsst.daf.base as dafBase
+import lsst.pex.logging as pexLog
+import lsst.afw.coord as afwCoord
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+
+class ObjectMaskCatalog(object):
+    """Class to support bright object masks
+
+    N.b. I/O is done by providing a readFits method which fools the (old) butler.
+    """
+    def __init__(self):
+        schema = afwTable.SimpleTable.makeMinimalSchema()
+
```

Comment:
(Picky, but...) why is the blank line here? It seems like it should either be removed or be a line lower (so that it appears between ``addField`` and ``self._catalog = ...``).